code/testapp/src/main/java/com/adobe/mobile/marketing/userprofile/testapp/MainActivity.kt

```diff
@@ -149,6 +165,32 @@ fun Input() {
                 )
             }
         }
+        Surface(
+            border = BorderStroke(1.dp, Color.Gray),
+            shape = RoundedCornerShape(8.dp),
+            modifier = Modifier.padding(8.dp)
+        ) {
+            Column(
+                modifier = Modifier.padding(PaddingValues(8.dp)),
+                horizontalAlignment = Alignment.CenterHorizontally
+            ) {
+                OutlinedTextField(
+                    value = appId,
+                    onValueChange = { appId = it },
+                    label = { Text("appId") }
+                )
+                Button(onClick = {
+                    MobileCore.configureWithAppID(appId)
+                }) {
+                    Text(text = "configureWithAppID(\"appId\")")
+                }
+            }
+        }
+        Button(onClick = {
+            MobileCore.trackAction("action", null)
```

Comment:
Can you update the action to MobileCore.trackAction("trigger_update_profile", null) in your launch property and add a similar comment:

To test this, configure a rule in your launch property that triggers a profile update for the following condition: a trackAction event with the action type 'trigger_update_profile'.

This way we will be able to maintain a single property for testing with different rule triggers.